tools/generate-release.sh

```diff
@@ -11,15 +11,21 @@ if [ ! -e "package.json" ]; then
   exit 1
 fi
 
-_remote=$(git remote get-url origin)
+npm install -g yarn
+
+git init .
+git config user.email "temporary@example.com"
+git config user.name "Temporary"
+git add .
+git commit -m "Temporary"
 git clone ./ gateway
-cd gateway
-git remote set-url origin "${_remote}"
+cd gateway || exit
```

Comment:
You may want this and line 28 to be `exit 1`.

Reply:
Good point! Error conditions should get error codes